### ISSUE TITLE
Can not use sequelize-typescript Models with sequelize ModelsHashInterface

### ISSUE THREAD

**Issue by chanlito** [RobinBuschmann/sequelize-typescript#200]
Hello everyone, currently `sequelize` instance has this `models` field which is of type `ModelsHashInterface`. 

Is there a better way than using (sequelize.models as MyOwnDefinedTypes) ?

**Comment by BruceHem** [comment]
```ts
export interface IModels {
  Todo: Model<Todo, any>; // Why I can't use typeof Todo here?
  [x: string]: Model<any, any>;
}

const models = sequelize.models as IModels;
```

Seems strange about that typings.

**Comment by RobinBuschmann** [comment]
You mean https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sequelize/index.d.ts#L5755 , right?

Unfortunately this is not compatible with the way sequelize-typescript defines its `Model`. So that `sequelize.Model !== sequelize-typescript.Model`. For `sequelize-typescript`it needs to look like:
```typescript
    interface ModelsHashInterface {
        [name: string]: typeof Model;
    }
```

**Comment by RobinBuschmann** [comment]
Ahh, and some 🎉🎉🎉🎉🎉 for the 200th issue 😀

**Comment by BruceHem** [comment]
Wow 200 issues already, didn't noticed it til u mentioned. 😄 🤣

**Comment by BruceHem** [comment]
@RobinBuschmann how would I override the existing type definition? I'm not experienced in this area. Been googling several hours now.

**Comment by RobinBuschmann** [comment]
Hmm, I see, overriding will not work. You will always get an `TS2415:Class 'XXX' incorrectly extends base class 'YYY'.
  Types of property 'aaa' are incompatible.
    Type 'ccc' is not assignable to type 'bbb'.` , right?

I'm afraid that we need to do the same with `Sequelize.d.ts` what we did with `Model.d.ts` to achieve this. So rewriting the whole typings for the Sequelize class.

**Comment by BruceHem** [comment]
😩  Guess I'll have to wait for next beta release.

**Comment by RobinBuschmann** [comment]
Fixed by 87cc694

**Comment by RobinBuschmann** [comment]
Released with `1.0.0-alpha.X`
